pyproject.toml

```diff
@@ -53,6 +54,13 @@ pytest = ">=6.0.0"
 pytest-cov = ">=2.6.1"
 codecov = "*"
 
+[tool.poetry.group.examples]
+optional = true
+
+[tool.poetry.group.examples.dependencies]
+jupyter = ">=1.0.0"
+pandas = ">=1.5.0"
+

```

Comment:
The build's failing because this group is now in the middel of the dev-dependencies, so poetry thinks everything below it (including meson and pybind) is optional, too. I suggest we move the examples group up, to be between `tqdm = "^4.61.1"` and `[tool.poetry.group.dev.dependencies]` (currently, that is line 50).

Reply:
Sorry, I missed that because of the blank line.